posthog/src/main/java/com/posthog/PostHogConfig.kt

```diff
@@ -46,23 +46,23 @@ public open class PostHogConfig(
      * Docs https://posthog.com/docs/feature-flags and https://posthog.com/docs/experiments
      * Defaults to true
      */
-    public var preloadFeatureFlags: Boolean = true,
+    public var preloadFeatureFlags: Boolean = false,
     /**
      * Number of minimum events before they are sent over the wire
      * Defaults to 20
      */
-    public var flushAt: Int = 20,
+    public var flushAt: Int = 100,
     /**
      * Number of maximum events in memory and disk, when the maximum is exceed, the oldest
      * event is deleted and the new one takes place
      * Defaults to 1000
      */
-    public var maxQueueSize: Int = 1000,
+    public var maxQueueSize: Int = 10000,
     /**
      * Number of maximum events in a batch call
      * Defaults to 50
      */
-    public var maxBatchSize: Int = 50,
+    public var maxBatchSize: Int = 100,
```

Comment:
values and comments are out of sync 🙈

Reply:
> Fix methods and classes comments (aka javadocs)

yes, its part of the TODO list on the PR description already